.github/workflows/foreman_plugin_js.yml

```diff
@@ -97,6 +100,20 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+      - name: store plugin name
+        run: echo "PLUGIN_NAME=$(echo ${{ matrix.plugin }} | awk -F'/' '{print $NF}')" >> "${GITHUB_ENV}"
+      - name: Set up plugin in Foreman
+        run: |
+          echo "gemspec name: '$PLUGIN_NAME', path: '${{ github.workspace }}/projects/plugin'" > "bundler.d/$PLUGIN_NAME.local.rb"
+          if [ -d $PLUGIN_NAME/gemfile.d ] ; then
+            cat $PLUGIN_NAME/gemfile.d/*.rb >> bundler.d/$PLUGIN_NAME.local.rb
+          fi
+        working-directory: ./foreman
+      - run: sudo apt-get update
+      - run: sudo apt-get -qq -y install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev libvirt-dev
```

Comment:
If you exclude the `libvirt` group I think `libvirt-dev` isn't needed. Having said that, I'd prefer to test as much as possible so I wouldn't exclude `journald` and `libvirt`.